src/graphql/main/schema.graphql

```diff
@@ -778,6 +779,14 @@ type OnChainUpdate {
   walletId: WalletId!
 }
 
+input OnChainUsdPaymentSendInput {
+  address: OnChainAddress!
+  amount: CentAmount!
+  memo: Memo
+  targetConfirmations: TargetConfirmations = 1
```

Comment:
I don't think I would expose this. eventually batching will expose more an enum, like `now` or `within 24 hours` or `within a week`.

Reply:
This one is in all our other send and probe mutations as well. We can probably remove it in a follow-up PR? (might be a schema change)